### PR TITLE
show share after update status

### LINKED ISSUE #279: Sharing screen is not showing up when updating status

### PR DESCRIPTION
**Pull request by diogoteix**
Closes #279

### CODE REVIEW

File: Trackovid19-web/src/app/screens/main/main.component.ts [hunk 1]
```diff
@@ -29,36 +29,31 @@ export class MainComponent implements OnInit, OnDestroy {
...
           this.showShare = true;
           this.renderer.setStyle(document.body, 'overflow', 'hidden');
           this.renderer.setStyle(document.body, 'position', 'fixed');
-          localStorage.setItem('share', 'true');
+          localStorage.setItem('share', 'false');
         }
       }
     });
 
+    let gdpr = localStorage.getItem('gdpr');
```
Comment (fabioscsilva) [inline Trackovid19-web/src/app/screens/main/main.component.ts:49]: Shouldn't this continue inside `ngOnInit`?

Comment (diogoteix) [reply Trackovid19-web/src/app/screens/main/main.component.ts:49]: It wasn't inside `ngOnInit`

Comment (diogoteix) [reply Trackovid19-web/src/app/screens/main/main.component.ts:49]: https://github.com/Tech4covid19/trackovid19-web/blob/develop/Trackovid19-web/src/app/screens/main/main.component.ts

**Review by fabioscsilva — APPROVED** [review]

**Review by nunofgm — APPROVED** [review]

File: Trackovid19-web/package-lock.json [hunk 2]
```diff
@@ -5425,8 +5425,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "resolved": false,
-          "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
```
Comment (fabioscsilva) [inline Trackovid19-web/package-lock.json]: Can you remove the updates on this file? You didn't updated the package.json, so you should not add this.

**Pull request merged**